security/security/src/main/java/io/helidon/security/DefaultAuditProvider.java

```diff
@@ -46,30 +66,40 @@ public static DefaultAuditProvider create(Config config) {
 
     private void audit(TracedAuditEvent event) {
         String tracingId = event.tracingId();
+        Level level;
```

Comment:
(No need to allocate the `tracingId` variable.)

Reply:
Moved to `logEvent` after checking if loggable.